src/bosh_azure_cpi/lib/cloud/azure/logger.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Bosh::AzureCloud
+  class CPILogger
+    def self.get_logger(device)
+      logger = Logging.logger(device,
```

Comment:
Why we implement a new logger instead of using `Bosh::Clouds::Config.logger`?

Reply:
Need to record thread id in logs, but the original logger (Logger) does not have such function.

Reply:
Thanks. Makes sense.